src/main/java/frc/robot/subsystems/ConditionedDriveSubsystem.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.usfirst.frc.team4999.utils.Utils;
+
+public class ConditionedDriveSubsystem extends SubsystemBase {
```

Comment:
To enable flexibility in changing the Drive stack, both ConditionedDriveSubsystem and the real DriveSubsystem should implement the same interface.  This allows the DriveCommand to not know which one it's actually talking to.  I can help you with this if it isn't obvious what to do.

Reply:
Or maybe break out a separate `DriveConditioner` interface, then have the `DriveCommand` feed values to the `DriveConditioner`, get the conditioned values, and feed the conditioned values to the `DriveSubsystem`?